api/models/participant.py

```diff
@@ -2,10 +2,12 @@
 from django.conf import settings
 from django.core import mail
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
But that's ridiculous. It's saying "Python's convention is A", "We've decided to ignore A and use A for something else so everybody who does Python will think that A means A but it means B".

The documentation states that the purpose is to: ta-ta-taaaa - - - **Save Typing** This convention is stupid. Saving typing is equivalent for obscuring the code. They wouldn't need to save typing if they did translation where it belongs - on the frontend.

When I look at "_", I see invalid variable name.